src/constants/VALIDATION.js

```diff
@@ -0,0 +1,13 @@
+const SIGN = Object.freeze({
+  REQUIRED_EMAIL: '이메일을 입력해 주세요.',
+  INVALID_EMAIL_FORMAT: '올바른 이메일 주소가 아닙니다.',
+  CHECK_EMAIL: '이메일을 확인해 주세요.',
+  EMAIL_REGEX: /^[a-zA-Z0-9+-\_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$/,
+
+  REQUIRED_PASSWORD: '비밀번호를 입력해 주세요.',
+  INVALID_PW_FORMAT: '8~20자 공백없이 영문/숫자/특수문자를 포함해 주세요.',
+  CHECK_PASSWORD: '비밀번호를 확인해 주세요.',
+  PW_REGEX: /^(?=.*[a-zA-Z])(?=.*[!@#$%^*+=-])(?=.*[0-9]).{8,20}$/,
```

Comment:
개인적으로 Regex의 경우 별도의 객체로 나눠서 관리하면 좋을 것 같아요.
텍스트는 텍스트끼리, 정규식은 정규식끼리 관리하면 둘의 관심사가 분리되어 유지보수하기 좋습니다!

```suggestion
const REGEX = {
  EMAIL_REGEX: /^[a-zA-Z0-9+-\_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$/,
  PW_REGEX: /^(?=.*[a-zA-Z])(?=.*[!@#$%^*+=-])(?=.*[0-9]).{8,20}$/,
  }
```

Reply:
안그래도 이 부분은 나눠야 될 거 같았는데 적용하도록 하겠습니다!